packages/grid/x-data-grid-pro/src/components/DataGridProVirtualScroller.tsx

```diff
@@ -307,9 +307,9 @@ const DataGridProVirtualScroller = React.forwardRef<
 
   // Create a lookup for faster check if the row is expanded
   const expandedRowIdsLookup = React.useMemo(() => {
-    const lookup: { [key: GridRowId]: boolean } = {};
+    const lookup: Set<GridRowId> = new Set();
```

Comment:
Although the `Set` is about [90% slower](https://jsperf.app/rupebo) than an object, I'd say it's fine because `expandedRowIdsLookup` is memorized and won't update quite often.

CC @cherniavskii

Reply:
I think it's fine, in Chrome it's only 40% slower.